Watchman.Discord.UnitTests/AntiSpam/SpamPunishmentStrategyTests.cs

```diff
@@ -77,17 +100,23 @@ public void SpamPunishmentStrategy_ShouldNotGiveAnyPunishment(int userWarnsCount
         public void SpamPunishmentStrategy_ShouldGiveMuteForTime(int userMutesCount, int expectedMinutesOfMute)
         {
             // Arrange
+            var warnsCount = 100;
```

Comment:
daj to jako const
choć wgl się zastanawiam czy jest sens tą liczbę przenosić do nowej zmiennej
@Marcin99b

Reply:
na razie zrobiłem tego consta